### PR TITLE
feat: add ability to use openid-connect flow for other providers

### LINKED ISSUE #2: setup ci
the following setting is wrong: https://github.com/siemens/sentry-auth-oidc/blob/master/.travis.yml#L17

### PR DESCRIPTION
**Pull request by max-wittig**
Fixes #2

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,29 +1,56 @@
...
+
+.. code-block:: python
+
+    CLIENT_ID = ""
+
+    CLIENT_SECRET = ""
+
+    SCOPE = "openid email"
+
+    WELL_KNOWN_URL = "https://accounts.google.com"  # e.g. for Google
```
Comment (bufferoverflow) [inline README.rst]: This is not the well known url, just to url of the OIDC server.

Maybe worth to add a link to https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig and mention that the configuration will be pulled from https://OIDC_URL/.well-known/openid-configuration

**Comment by max-wittig** [comment]
@bufferoverflow Will add travis-ci releases and then we can merge this

**Comment by max-wittig** [comment]
@bufferoverflow Sadly we cannot publish every commit on the test PyPI, because we would need to increase the version, which would be confusing with the real PyPI release.

File: README.rst (hunk 1 above)
Comment (bufferoverflow) [reply README.rst]: @max-wittig Maybe rename WELL_KNOWN_URL to OIDC_DOMAIN or so in a next commit, as this URL is not the WELL_KNOWN_URL.

**Pull request merged**